src/coreclr/nativeaot/Test.CoreLib/src/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="..\..\Directory.Build.props" />
+  <ItemGroup>
+    <EditorConfigFiles Remove="$(RepositoryEngineeringDir)CodeAnalysis.src.globalconfig" />
+    <EditorConfigFiles Include="$(RepositoryEngineeringDir)CodeAnalysis.test.globalconfig" />
+  </ItemGroup>
+</Project>
```

Comment:
With this change, why do we need the subsequent changes under Test.CoreLib?  The next two files in the diff have a warning suppression for IDE0060, but that shouldn't be enabled in CodeAnalysis.test.globalconfig.